docs/install.md

```diff
@@ -241,7 +237,23 @@ public $filters = [
 ];
 ```
 
-> **Note** If you have grouped or changed the default format of the routes, ensure that your code matches the new format(s) in the **app/Config/Filter.php** file.
```

Comment:
Why do you remove the note?

Reply:
I didn't remove the note, just pushed it down. 

The other corrections you pointed out were a result of the rebasing I did. I guess I didn't pay attention to the files properly. 
Thanks for taking the time to check.

Reply:
Thank you for fixing the docs. It seems good now.

I think `git rebase` dose not change indentation. But if you got conflicts,
you might change unexpected changes by accident when resolving conflicts.

I recommend you check the diff on GitHub PR page. Sometimes I notice unexpected changes.